index.html

```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0"> 
+  <link rel="icon" type="image/png" sizes="32x32" href="images/favicon-32x32.png">
+  <link rel="stylesheet" href="styles.css">
+  <title>Frontend Mentor | QR code component</title>
+</head>
+<body>
+  <section class="main-section">
```

Comment:
Great HTML structure.👏 I suggest you could use a `main` tag for the whole section instead of the `section` tag since there is only one section. Section tags are good for dividing a page into different parts. So since you have only one card, a `main` tag is better for semantics purposes.

Reply:
Wow! Thank you I will definitely do that before merging